app/transport/http/user.go

```diff
@@ -41,7 +41,7 @@ func (uh UserHandler) CreateUser() http.Handler {
 				return
 			}
 
-			WriteJSONResponse(w, http.StatusInternalServerError, Response{MsgInternalSeverErr}, uh.logger)
+			WriteJSONResponse(w, http.StatusInternalServerError, Response{MsgInternalServerErr + err.Error()}, uh.logger)
```

Comment:
Messages of server errors are usually not exposed in the responses. Because it can't help a client anyway while the error message might contain something you don't want a client to see.

Reply:
> 

I got it, thanks.

Reply:
> Messages of server errors are usually not exposed in the responses. Because it can't help a client anyway while the error message might contain something you don't want a client to see.

I got it, thanks.

Reply:
I got it, thanks.